src/pages/Admin/Charity/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,100 @@
+import { PropsWithChildren } from "react";
+import { useAdminContext } from "pages/Admin/Context";
+import { useProfileQuery } from "services/aws/aws";
+import ContentLoader from "components/ContentLoader";
+import QueryLoader from "components/QueryLoader";
+import Seo from "../Seo";
+import Balance from "./Balance";
+
+export default function Dashboard() {
+  const { id } = useAdminContext();
+  const queryState = useProfileQuery({ endowId: id }, { skip: !id });
+
+  return (
+    <div className="@container w-full max-w-4xl grid content-start mt-6">
+      <Seo title="Endowment Dashboard" />
+      <h3 className="uppercase font-extrabold text-2xl mb-4">Dashboard</h3>
+      <QueryLoader
+        queryState={queryState}
+        messages={{
+          loading: <LoaderSkeleton />,
+          error: "Failed to get endowment info",
+        }}
+      >
+        {({
+          contributionsCount,
+          donationsBal,
+          payoutsPending,
+          payoutsMade,
+          sustainabilityFundBal,
+          totalEarnings,
+          totalContributions,
+        }) => {
+          const earningsPayoutsPending = payoutsPending - donationsBal;
+          const earningsPayoutsMade = totalEarnings - earningsPayoutsPending;
+          return (
+            <div className="grid gap-4 @lg:grid-cols-2">
+              {/** Header */}
+              <div className="col-span-2 grid @2xl:grid-cols-3 gap-4">
+                <HeaderPart>
+                  Pending payout:
+                  <span className="flex items-center gap-1">
+                    {payoutsPending}{" "}
+                    <span className="text-2xs sm:text-xs font-normal">USD</span>
+                  </span>
+                </HeaderPart>
+                <HeaderPart>
+                  Total Contributions:
+                  <span className="flex items-center gap-1">
+                    {totalContributions}{" "}
+                    <span className="text-2xs sm:text-xs font-normal">USD</span>
+                  </span>
+                </HeaderPart>
+                <HeaderPart>
+                  Contribution Count:
+                  <span>{contributionsCount}</span>
+                </HeaderPart>
+              </div>
+
+              {/** Balances */}
+              <Balance
+                type="Donations"
+                current={donationsBal}
+                pending={donationsBal}
```

Comment:
This should be current payouts pending I think?

Reply:
@SovereignAndrey for donations the current balance and amount pending payout should always be the same, right?

Reply:
@misicnenad Yes, you're correct! Just the super murky intent/terminology that I keep thinking it's a bug when I see those lines. :sweat_smile: